Generate periodic statistics at client

dominiklohmann opened the pull request.

dominiklohmann commented:
[story/ch9335]

mavam commented on scripts/vast-du:
```diff
@@ -0,0 +1,23 @@
+#! /usr/bin/env sh
```
I think a shebang of `#!/bin/sh` is also POSIX compliant.

mavam commented on scripts/vast-du:
```diff
@@ -0,0 +1,23 @@
+
+if ! command -v vast >/dev/null 2>&1; then
+	echo "vast not found"
+	exit
```
```suggestion
	exit 1
```

dominiklohmann commented:
I’ve addressed the feedback. The script isn’t quite up to the usual standard, because it’s just what I used for testing locally and we decided it was worth keeping.
